synthtool/gcp/templates/python_library/.github/CODEOWNERS

```diff
@@ -0,0 +1,19 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+{% if 'codeowner_team' in metadata['repo'] %}
```

Comment:
Looks like the BigQuery repos are missing this property. Are you planning on sending some PRs to update the repo metadata?

Reply:
Yes, I also need to add `default_version` to `.repo-metadata.json` for #1114. I'm planning to add it at the same time. I see nodejs already uses `default_version` so I'm following their lead. https://github.com/googleapis/nodejs-iap/blob/main/.repo-metadata.json#L12

Reply:
Done.